### PR TITLE
Enable vision

### PR DESCRIPTION
**Pull request by james-ward**

### CODE REVIEW

File: components/vision.py [hunk 1]
```diff
@@ -99,42 +99,41 @@ def execute(self) -> None:
...
+        if self.fuse_vision_observations:
+            innovation = vision_pose.translation().distance(
+                self.chassis.estimator.getEstimatedPosition().translation()
+            )
+            # Gate on innovation
+            if innovation > 1.0:
+                return
+            # Come up with a position std dev from the angle in the picture
+            # When the target is near the edge, the estimate of range is worse
+            pos_std_dev = 0.2 + 0.4 * abs(self.vision_data.angle) / math.radians(15)
```
Comment (OliverW10) [inline components/vision.py:130]: do we want to incorporate the confidence returned from vision in this std dev?

Comment (james-ward) [reply components/vision.py:130]: I thought about that, but it isn't actually in a scale that is useful. There is no way from looking at the vision code to work out what the range of values it returns are, so I ignored it and did it this way.
Ideally the vision system should return a confidence for the values it returns, and these should be in the same units as the observations. @LucienMorey @whert-dev ?

Comment (james-ward) [reply components/vision.py]: Okay, so now it uses the fitness (which is in the range [0,1] to derive a std dev.

File: components/vision.py [hunk 2]
```diff
@@ -45,7 +42,8 @@ class Vision:
     shooter: Shooter
```
Comment (OliverW10) [inline components/vision.py:42]:
```suggestion
```

Comment (OliverW10) [reply components/vision.py:42]: imu too

Comment (james-ward) [reply components/vision.py]: Good catch. Done.

**Pull request merged**